rel/value.go

```diff
@@ -108,12 +108,25 @@ type Set interface {
 	Without(Value) Set
 	Map(func(Value) Value) Set
 	Where(func(Value) bool) Set
-	Call(arg Value) Value
 	CallAll(arg Value) Set
 
 	ArrayEnumerator() (OffsetValueEnumerator, bool)
 }
 
+// SetCall does a CallAll to a Set and panics when there's less or more than 1 value.
+func SetCall(s Set, arg Value) Value {
+	result := s.CallAll(arg)
+	if !result.IsTrue() {
+		panic("no result")
+	}
+	if result.Count() > 1 {
```

Comment:
Avoid `Count` where possible, since it's O(&infin;) worst case. Enumerate up to two elements and fail if you reach the second one.